net-analyzer/nuttcp/Manifest

```diff
@@ -0,0 +1 @@
+DIST nuttcp-8.1.4.tar.bz2 13063356 SHA256 5c5f4f6ae04adb8a86d11e1995939c1308b90e1946ebc77c9988b5eb85961bb5 SHA512 ba6ff33bc1790c92f47d760d4588b128f60f0ca09ed43d95fd10b0edfb636a493b0d26b195e688a13b5b79fafc1cc52a0c73455263f921ff8ad0cf087f1d937b WHIRLPOOL c8efd3ec1fcaed072cae48c70dfea136d972372730896154ac1ff6fd6b4e594b24186bdbe704fbb39db1c70c544417755174051133dc1cc710a8c175f230b249
```

Comment:
Please update the Manifest to use current hash set (i.e. remove the file and run `repoman manifest`).

Reply:
Done.

Reply:
Either 'nope', or you forgot to push ;-). I still see SHA256 and WHIRLPOOL.